openc3/lib/openc3/models/cvt_model.rb

```diff
@@ -165,19 +188,24 @@ def self._parse_item(lookups, item)
       target_name, packet_name, item_name, value_type = item.split('__')
       raise ArgumentError, "items must be formatted as TGT__PKT__ITEM__TYPE" if target_name.nil? || packet_name.nil? || item_name.nil? || value_type.nil?
 
-      # We build lookup keys by including all the less formatted types to gracefully degrade lookups
-      # This allows the user to specify WITH_UNITS and if there is no conversions it will simply return the RAW value
-      case value_type.upcase
-      when 'RAW'
-        keys = [item_name]
-      when 'CONVERTED'
-        keys = ["#{item_name}__C", item_name]
-      when 'FORMATTED'
-        keys = ["#{item_name}__F", "#{item_name}__C", item_name]
-      when 'WITH_UNITS'
-        keys = ["#{item_name}__U", "#{item_name}__F", "#{item_name}__C", item_name]
+      if @overrides["#{target_name}__#{packet_name}__#{item_name}__#{value_type}"]
+        # Set the result as a Hash to distingish it from the key array and from an overridden Array value
+        keys = {'value' => @overrides["#{target_name}__#{packet_name}__#{item_name}__#{value_type}"]}
```

Comment:
I don't actually care that it's a Hash but it can't be single value or an Array so this was an easy check